salt-mode.el

```diff
@@ -54,12 +54,23 @@
 ;; C-c % %
 ;;   for {# and {% as well.
 
+;; In-Emacs documentation: If you have Salt installed on the same
```

Comment:
No, this is just a continuation of the commentary section, describing the feature. Weirdly, [commentary is hidden for third-party packages](https://www.reddit.com/r/emacs/comments/4ise49/describepackage_commentary_for_non_builtin/), but I only noticed this a few weeks ago and didn't have time to follow it up more yet.

But thinking more, this is probably too important for the commentary section. I'll move some version of it into the doc string for the mode.